src/api/app/views/webui/main/index.html.haml

```diff
@@ -8,9 +8,7 @@
       .card-body
         %h2
           Welcome to #{@configuration['title']}
-        -# rubocop:disable Rails/OutputSafety
-        = raw @configuration['description']
-        -# rubocop:enable Rails/OutputSafety
+        = sanitize @configuration['description']
```

Comment:
Why do you sanitize this now?

Reply:
To align behaviours on how we render similar fields.

Reply:
I see two options:
1. Use `sanitize @configuration[...]` in both files: `app/views/webui/main/index.html.haml` and `app/views/webui/code_of_conduct/index.html.haml`, or
2. Use `raw @configuration[...]`  in both files: `app/views/webui/main/index.html.haml` and `app/views/webui/code_of_conduct/index.html.haml`

Do you mean you prefer option _2._?

Reply:
> I prefer giving OBS admins safe and sane ways of modifying these fields.

Safe from themselves?

Reply:
> Safe from themselves?

Sure.

And I still can't imagine a valid counter case.

Reply:
Just because you can't imagine one doesn't mean it doesn't exist. But okay, we can sanitize if you want to.